runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/FlattenStreamingTest.java

```diff
@@ -80,4 +90,35 @@ public void testRun() throws Exception {
     res.close();
   }
 
+  @Test
+  public void testFlattenBoundedUnbounded() throws Exception {
```

Comment:
The runner has timeout and can look at PAssert aggregators, but those seem to be ignored now with `PAssertStreaming` (though they're still there).
Once `Read.Bounded` and `Read.Unbounded` are in (they're down the pipeline after this one), it would be a good place to stop and see which streaming tests are supportted and could be removed from the runner (batch _should_ be supported completely after `Read.Bounded` is supported).